### PR TITLE
Add document icon

### PR DESCRIPTION
**Pull request by jmophoto**
Add document to the Icon component.

Usage
```<Icon type='document' />```

screen shot 2016-02-17 at 9 30 28 am

@mxenabled/frontend-engineers

### CODE REVIEW

**Comment by paniclater** [comment]
:shipit:

**Comment by jmophoto** [comment]
@bsbeeks walked me through merging the paths for this and I pushed up that change. 

There was also a request to change the export icon to be import to match the design. I have added the import icon, but left the export one so it isn't a breaking change. I have also added a warning to be consoled anytime the export icon is used. Thoughts on this pattern for deprecation?

@mxenabled/frontend-engineers

File: src/components/Icon.js [hunk 1]
```diff
@@ -318,6 +325,7 @@ const Icon = React.createClass({
             l196.1,151.7c2.6,2,5.7,3,8.8,3c3.2,0,6.5-1.1,9.1-3.3L457,147.3V376C457,380.2,453.5,383.8,449.4,383.8z' />
         );
       case 'export':
+        console.error('WARNING: The export icon has been moved to import and will be removed or changed in a future release.');
```
Comment (bsbeeks) [inline src/components/Icon.js]: let's do a console.warn instead and put quotes around export:

```console.warn('WARNING: The "export" icon has been replaced with "import" and will be remove in a future release.")```

Comment (jmophoto) [inline src/components/Icon.js]: :+1: 
screen shot 2016-02-17 at 10 15 07 am

**Comment by cerinman** [comment]
:shipit: Will release this with a few of my changes under version `2.0.0-rc.3`

**Pull request merged**